libraries/Microsoft.Bot.Connector/Authentication/BotFrameworkAuthentication.cs

```diff
@@ -49,6 +50,15 @@ public abstract class BotFrameworkAuthentication
         /// <returns>Asynchronous Task with <see cref="UserTokenClient" /> instance.</returns>
         public abstract Task<UserTokenClient> CreateUserTokenClientAsync(ClaimsIdentity claimsIdentity, CancellationToken cancellationToken);
 
+        /// <summary>
+        /// Creates a <see cref="BotFrameworkClient"/> used for calling Skills.
+        /// </summary>
+        /// <returns>A <see cref="BotFrameworkClient"/> instance to call Skills.</returns>
+        public virtual BotFrameworkClient CreateBotFrameworkClient()
```

Comment:
I find the location of this method odd, I would have never guessed that a class named BofrFrameworkAuthentication would be responsible of creating a BotFrameworkClient instance (I'll keep reviwing and update this comment if I can find of a better place.)